test/test-debuglet.ts

```diff
@@ -289,16 +291,19 @@ describe('Debuglet', function() {
   });
 
   describe('setup', function() {
-    before(function() { oldGP = process.env.GCLOUD_PROJECT; });
+    before(function() {
+      oldGP = process.env.GCLOUD_PROJECT;
+    });
 
     after(function() { process.env.GCLOUD_PROJECT = oldGP; });
-
-    beforeEach(function() {
+    beforeEach(() => {
       delete process.env.GCLOUD_PROJECT;
       nocks.oauth2();
     });
 
-    afterEach(function() { nock.cleanAll(); });
+    afterEach(() => {
```

Comment:
nit: For consistency with the rest of the file, we could use `function() {}` instead of `() => {}`.

Reply:
Done.